carmajava/rsumetering/src/main/java/gov/dot/fhwa/saxton/carma/rsumetering/CommandingState.java

```diff
@@ -32,6 +32,15 @@
  * Set the target vehicle speed to the platoon speed and activate the lane change indicator when needed
  */
 public class CommandingState extends RSUMeteringStateBase {
+  protected static final double MIN_METER_DIST = 0.0; //meters
```

Comment:
again we should add some buffer space here of -10 m

Reply:
done